site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: doc
+title: Integrate Your Ruby on Rails API With FusionAuth
+description: Integrate your Ruby on Rails API with FusionAuth
+navcategory: getting-started
+prerequisites: Ruby, bundler and Rails
+technology: Ruby on Rails
+language: Ruby
+---
+
+## Integrate Your {{page.technology}} API With FusionAuth
+
+{% include docs/integration/_intro-api.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we’re going to use the [{{page.language}} client library](/docs/v1/tech/client-libraries/ruby). You can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Then, create the required files:
+
+```shell
+touch Gemfile
+```
+
+Now, copy and paste the following file into `Gemfile`.
+
+```ruby
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/ruby/Gemfile %}
+
+Install the gems.
+
+```shell
+bundle install
+```
+
+Then copy and paste the following code into the `setup.rb` file.
+
+```ruby
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/ruby/setup-api.rb %}
+```
+Then, you can run the setup script.
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the [Ruby client library](/docs/v1/tech/client-libraries/ruby) documentation for more information." %}
+
+This will create the FusionAuth configuration for your {{page.technology}} API.
+
+```shell
+fusionauth_api_key=YOUR_API_KEY_FROM_ABOVE ruby setup.rb
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='YOUR_API_KEY_FROM_ABOVE'
+ruby setup.rb
+```
+
+If you want, you can http://localhost:9011[login to your instance] and examine the new API configuration the script created for you. 
+
+## Create Your {{page.technology}} API
+
+Now you are going to create a {{page.technology}} API. While this section builds a simple {{page.technology}} API, you can use the same configuration to build a more complex {{page.technology}} API.
+
+First, create the skeleton of the {{page.technology}} API. Rails has a nice generator to build this out.
+
+```shell
+rails new myapi --api && cd myapi
+```
+
+Now, create a new `Gemfile` file.
```

Comment:
we did provide explicit `touch` and `copy` instructions for this earlier. Rails already creates this one. This file also may bump with the ruby version (I was running 2.7.5 running through this but the included one references 3.2.2. Looks like we're just adding deps.